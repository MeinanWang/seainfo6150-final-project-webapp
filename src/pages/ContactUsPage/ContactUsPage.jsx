import React from 'react'
import styles from './ContactUsPage.module.css';

import PageHeader from '../../reuseableComponents/PageHeader/PageHeader';
import { webContactInfo } from '../../data';

const ContactUsPage = () => {
    return (
        <div>
            <PageHeader header='Contact Us' />
            <ul className={styles.container}>
                {webContactInfo.map((info, index) => (
                    <li key={index} className={styles.infoCard}>
                        <h3>{info.name}</h3>
                        <p>{info.address}</p>
                        <p>Phone: {info.phone}</p>
                        <p>Fax: {info.fax}</p>
                        <p>Email: {info.email}</p>
                    </li>
                ))}
            </ul>
        </div>
    )
};

export default ContactUsPage;
