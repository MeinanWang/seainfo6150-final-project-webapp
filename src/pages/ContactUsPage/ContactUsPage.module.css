.container{
    margin: 0 20%;
}

.infoCard{
    margin-top: 40px;
}