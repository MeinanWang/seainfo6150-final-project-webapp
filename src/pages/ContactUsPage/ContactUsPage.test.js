import React from "react";
import { render } from "@testing-library/react";
import ContactUsPage from "./ContactUsPage.jsx";

it("renders the Contact Us page correctly", () => {
    const { container } = render(<ContactUsPage />);
    expect(container).toMatchSnapshot();
});
