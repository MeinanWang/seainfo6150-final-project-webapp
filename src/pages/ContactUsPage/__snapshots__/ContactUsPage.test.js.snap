// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Contact Us page correctly 1`] = `
<div>
  <div>
    <div
      class="container"
    >
      <div
        class="heading"
      >
        <h1
          class="name"
        >
          Contact Us
        </h1>
      </div>
    </div>
    <ul
      class="container"
    >
      <li
        class="infoCard"
      >
        <h3>
          Registered Office
        </h3>
        <p>
          Recipe press road, Sunnyvale, CA
        </p>
        <p>
          Phone: 
          +123 456 7890
        </p>
        <p>
          Fax: 
          +123 456 7890
        </p>
        <p>
          Email: 
          info@recipepress.com
        </p>
      </li>
      <li
        class="infoCard"
      >
        <h3>
          Our Restaurant
        </h3>
        <p>
          Recipe press road, Sunnyvale, CA
        </p>
        <p>
          Phone: 
          +123 456 7890
        </p>
        <p>
          Fax: 
          +123 456 7890
        </p>
        <p>
          Email: 
          info@recipepress.com
        </p>
      </li>
      <li
        class="infoCard"
      >
        <h3>
          Other Office
        </h3>
        <p>
          Recipe press road, Sunnyvale, CA
        </p>
        <p>
          Phone: 
          +123 456 7890
        </p>
        <p>
          Fax: 
          +123 456 7890
        </p>
        <p>
          Email: 
          info@recipepress.com
        </p>
      </li>
    </ul>
  </div>
</div>
`;
